src/app/controllers/components/ControllerHeaderForms/AddController/AddController.test.tsx

```diff
@@ -25,38 +24,53 @@ describe("AddController", () => {
           { name: ConfigNames.RPC_SHARED_SECRET, value: "veryverysecret" },
         ],
       }),
+      general: generalStateFactory({
+        version: versionStateFactory({ data: "3.2.0" }),
+      }),
     });
   });
 
   it("includes the config in the instructions", () => {
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter>
-          <AddController clearHeaderContent={jest.fn()} />
-        </MemoryRouter>
-      </Provider>
-    );
-    const instructions = wrapper
-      .find("div[data-testid='register-snippet']")
-      .text();
-    expect(instructions.includes("http://1.2.3.4/MAAS")).toBe(true);
-    expect(instructions.includes("veryverysecret")).toBe(true);
+    renderWithBrowserRouter(<AddController clearHeaderContent={jest.fn()} />, {
+      wrapperProps: { state },
+    });
+    const instructions = screen.getByTestId("register-snippet");
+    expect(
+      within(instructions).getByText(new RegExp("http://1.2.3.4/MAAS"))
+    ).toBeInTheDocument();
+    expect(
+      within(instructions).getByText(/veryverysecret/)
+    ).toBeInTheDocument();
   });
 
-  it("can close the instructions", () => {
-    const store = mockStore(state);
+  it("can close the instructions", async () => {
     const clearHeaderContent = jest.fn();
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter>
-          <AddController clearHeaderContent={clearHeaderContent} />
-        </MemoryRouter>
-      </Provider>
+    renderWithBrowserRouter(
```

Comment:
It probably doesn't make much of a difference, but I don't think these need the router test wrapper.